app-cdr/iat/iat-0.1.7-r1.ebuild

```diff
@@ -1,15 +1,15 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 DESCRIPTION="BIN, MDF, PDI, CDI, NRG, and B5I converters"
-HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
-SRC_URI="mirror://gentoo/${P}.tar.bz2"
-
+HOMEPAGE="https://www.berlios.de/software/iso9660-analyzer-tool"
+SRC_URI="https://download.sourceforge.net/iat.berlios/${P}.tar.bz2"
```

Comment:
Soo have you verified this is indeed the same file as hosted in gentoo mirrors?

Reply:
The manifest didn't change, so I would say, yes.
Just for thoroughness, I downloaded them both and compared the `md5sum` output:

```sh
5e1ffd82000ea4c80ffb2d63ae55120c gentoo/iat-0.1.7.tar.bz2
5e1ffd82000ea4c80ffb2d63ae55120c sourceforge/iat-0.1.7.tar.bz2
```